crates/viona-api/sys/src/lib.rs

```diff
@@ -67,6 +68,14 @@ mod structs {
         pub vrs_qsize: u16,
         pub vrs_qaddr: u64,
     }
+
+    #[repr(C)]
+    pub enum vioc_promisc {
```

Comment:
This should be renamed to `viona_promisc_t` to match the enum in `viona_io.h`?

Reply:
I'm happy to do that, but all of the other definitions in this file omit the `_t`.  It's not a direct comparison, as the others are `struct`s rather than `enum`s, but leaving off the `_t` feels like a better fit.

Reply:
It's `viona_promisc_t`, not `vioc_promisc_t`.

Reply:
Sheesh.  That's embarrassing.  I had the two files open in side-by-side windows and couldn't see it.  Sorry for the clumsy oversight.

Reply:
No worries.  One could argue that the name for that enum should be made consistent with the other structs in that header, considering the other names.